<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BenisGUI</class>
 <widget class="QMainWindow" name="BenisGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hide my Benis!</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>12</y>
      <width>561</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>General settings</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="browseInputImage">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="inputImage"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="passwordInput"/>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="fastMode">
       <property name="text">
        <string>Enable fast-mode</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Hide settings</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="browseInputFile">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="inputFile"/>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="browseOutputImage">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="outputImage"/>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="hideButton">
       <property name="text">
        <string>Hide</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Extraction settings</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QPushButton" name="browseOutputFile">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="outputFile"/>
     </item>
     <item row="12" column="1">
      <widget class="QPushButton" name="extractButton">
       <property name="text">
        <string>Extract</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
